bin/install-wp-tests.sh

```diff
@@ -23,8 +23,15 @@ download() {
     fi
 }
 
-if [[ $WP_VERSION =~ [0-9]+\.[0-9]+(\.[0-9]+)? ]]; then
-	WP_TESTS_TAG="tags/$WP_VERSION"
+if [[ $WP_VERSION =~ ^[0-9]+\.[0-9]+$ ]]; then
+	WP_TESTS_TAG="branches/$WP_VERSION"
+elif [[ $WP_VERSION =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+	if [[ $WP_VERSION =~ [0-9]+\.[0-9]+\.[0] ]]; then
```

Comment:
this code is duplicated below in `install_wp()`. it looks like all _this_ section should be doing is deciding whether the `WP_TESTS_TAG` should use `branches/` or `tags`.

Reply:
since the `$WP_TESTS_TAG` variable is only used in the `install_test_suite()` function, you could also move this section into there, as opposed to leaving it in the main body.